docs/references.txt

```diff
@@ -1,4 +1,3 @@
-.. _Astropy: http://astropy.org
```

Comment:
Yes -- I'm still mystified as to why RST doesn't easily allow this kind of thing.

Reply:
Sorry, what's the reason this stopped working? And why would the other urls still be working?

Reply:
It builds without removing this but generates warnings, and the warning were promoting to errors, so I moved the definition inline everywhere. I believe none of the other names defined here show up in more than one place.  An alternative would be to revert the change in links and change the documentation build so that warnings are not treated as errors.